app/root/header/Header.tsx

```diff
@@ -100,119 +128,123 @@ export function Header() {
-              >
-                Notices
-              </NavLink>,
-              <NavLink
-                className="usa-nav__link"
-                to="/circulars"
-                key="/circulars"
-                onClick={hideMobileNav}
-              >
-                Circulars
-              </NavLink>,
-              <NavLink
-                className="usa-nav__link"
-                to="/docs"
-                key="/docs"
-                onClick={hideMobileNav}
-              >
-                Documentation
-              </NavLink>,
-              email ? (
-                <>
-                  <NavDropDownButton
-                    to="/user"
-                    type="button"
-                    key="user"
-                    label={email}
-                    isOpen={userMenuIsOpen}
-                    onToggle={() => {
-                      setUserMenuIsOpen(!userMenuIsOpen)
-                    }}
-                    menuId="user"
-                  />
-                  <Menu
-                    id="user"
-                    isOpen={userMenuIsOpen}
-                    items={[
-                      <NavLink
-                        end
-                        key="user"
-                        to="/user"
-                        onClick={onClickUserMenuItem}
-                      >
-                        Profile
-                      </NavLink>,
-                      <NavLink
-                        key="endorsements"
-                        to="/user/endorsements"
-                        onClick={onClickUserMenuItem}
-                      >
-                        Peer Endorsements
-                      </NavLink>,
-                      !idp && (
-                        <NavLink
-                          key="password"
-                          to="/user/password"
-                          onClick={onClickUserMenuItem}
-                        >
-                          Reset Password
-                        </NavLink>
-                      ),
-                      <NavLink
-                        key="credentials"
-                        to="/user/credentials"
-                        onClick={onClickUserMenuItem}
-                      >
-                        Client Credentials
-                      </NavLink>,
-                      <NavLink
-                        key="email"
-                        to="/user/email"
-                        onClick={onClickUserMenuItem}
-                      >
-                        Email Notifications
-                      </NavLink>,
-                      <Link
-                        key="logout"
-                        to="/logout"
-                        onClick={onClickUserMenuItem}
-                      >
-                        Sign Out
-                      </Link>,
-                    ]}
-                  />
-                </>
-              ) : (
-                <Link
+          <div ref={menuRef}>
```

Comment:
Can this ref be moved to the PrimaryNav component instead of wrapping it in a div? 

The div adds some extra styling that bunches up the sign in button:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/881b3495-af8d-4b42-a387-92751ea37851)

Reply:
Actually this can just be fixed by adding the `.text-no-wrap` class to the button, you can ignore my previous comment

Reply:
Just updated to fix